### ISSUE TITLE
[BUG] Playwright install fails on windows 10 

### ISSUE THREAD

**Issue by karthikdevarakonda** [microsoft/playwright#5636]
**Context:**
- Playwright Version: [1.9.1]
- Operating System: [Windows]
- Node.js version: [v14.16.0]
- Browser: [e.g. All, Chromium, Firefox,]

Getting the following error 

npm install -D playwright

> playwright@1.9.1 install <Directory>\node_modules\playwright
> node install.js

(node:23000) UnhandledPromiseRejectionWarning: Error: Failed to download chromium, caused by
Error: unable to get local issuer certificate
    at TLSSocket.onConnectSecure (_tls_wrap.js:1497:34)
    at TLSSocket.emit (events.js:315:20)
    at TLSSocket._finishInit (_tls_wrap.js:932:8)
    at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:706:12)
    at <Directory>\node_modules\playwright\lib\install\installer.js:133:19
    at async validateCache (<Directory>\node_modules\playwright\lib\install\installer.js:132:9)
    at async installBrowsersWithProgressBar (<Directory>\\node_modules\playwright\lib\install\installer.js:82:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:23000) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:23000) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will
terminate the Node.js process with a non-zero exit code.

**Comment by pavelfeldman** [comment]
Your system does not trust Microsoft CDN certificate. Are you behind a corporate proxy or firewall?

**Comment by karthikdevarakonda** [comment]
Yes its behind a firewall, Tried to set the Strict ssl to false even then receiving this error

**Comment by aslushnikov** [comment]
We updated [installation docs](https://playwright.dev/docs/installation?_highlight=insta#install-behind-the-firewall-or-proxy) - please take a look. It should address all the firewall-related questions.

If the questions remain - please file a new issue!

**Comment by jmeachum** [comment]
I ran into this same issue today. My issue is that we have an internally signed ca which is not trusted by node. So the download of playwright would work but the node install.js would fail with the error above. In my case I installed python package so the install.js file was still on my host. I set SSL_CERT_FILE env variable to the path of the internal ca.pem file, and used the node --use-openssl-ca flag which then downloaded the files.
